examples/light-switch-app/asr/include/BindingHandler.h

```diff
@@ -52,9 +52,11 @@ class BindingHandler
     }
 
 private:
```

Comment:
This seems like a rough API incompatibility

Could we document it in upgrading.md? Did we consider what it would take to not break the APIs?